app/models/user.rb

```diff
@@ -21,4 +21,5 @@ def free_balance
 
   delegate :balance_transactions, to: :transactions
   delegate :category_transactions, to: :transactions
+  delegate :categories_have_budget, to: :categories
```

Comment:
@marderer maybe you would use `prefix: true` for delegation? Check docs about delegate

```ruby
delegate :with_amount, to: :categories, prefix: true 
```